### PR TITLE
Add configurability of tooltip position relative to cursor

### PR DESCRIPTION
**Pull request by Crypthead**
Allows changing the position of the tooltip to be under (default) or above the cursor depending on the configuration.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java [hunk 1]
```diff
@@ -276,4 +276,15 @@ default int infoBoxSize()
 	{
 		return 35;
 	}
+
+	@ConfigItem(
+		keyName = "tooltipPosition",
+		name = "Tooltip Position",
+		description = "Configures whether to show the tooltip above or under the cursor.",
+		position = 35
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java]: The config items should be sorted by position

File: runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java [hunk 2]
```diff
@@ -81,7 +86,8 @@ private Dimension renderTooltips(Graphics2D graphics, List<Tooltip> tooltips)
 	{
 		final Rectangle clientCanvasBounds = new Rectangle(client.getRealDimensions());
 		final net.runelite.api.Point mouseCanvasPosition = client.getMouseCanvasPosition();
-		final Point mousePosition = new Point(mouseCanvasPosition.getX(), mouseCanvasPosition.getY() + OFFSET);
+		final int offset = ((runeLiteConfig.tooltipPosition() == TooltipPositionType.UNDER_CURSOR) ? UNDER_OFFSET : ABOVE_OFFSET);
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java]:
```suggestion
		final int offset = runeLiteConfig.tooltipPosition() == TooltipPositionType.UNDER_CURSOR ? UNDER_OFFSET : ABOVE_OFFSET;
```

File: runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java (hunk 1 above)
Comment (Crypthead) [reply runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java]: Sorted

File: runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java (hunk 2 above)
Comment (Crypthead) [reply runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java]: Applied changes

**Comment by Crypthead** [comment]
Applied all recommended changes

**Pull request merged**